app/views/foreman_tasks/tasks/dashboard/_task_status_table.html.erb

```diff
@@ -0,0 +1,14 @@
+<table class="table table-striped">
+  <tr>
+    <th><%= _("State") %></th>
+    <th><%= _("Result") %></th>
+    <th><%= _("No. of Tasks") %></th>
+  </tr>
+  <% ForemanTasks::Task::Summarizer.new.summarize_by_status.each do |result| %>
+    <tr>
+      <td><%= result.state %></td>
+      <td><%= result.result %></td>
+      <td><%= link_to result.count, main_app.foreman_tasks_tasks_path(:search => "state=#{result.state}&result=#{result.result}") %></td>
```

Comment:
Suggestion: some visual indication of erroneous combinations (especially 'paused' state and 'error' or 'warning' result) might be useful here